src/pages/home/report/ReportActionsList.tsx

```diff
@@ -207,9 +206,7 @@ function ReportActionsList({
     );
     const prevSortedVisibleReportActionsObjects = usePrevious(sortedVisibleReportActionsObjects);
 
-    const reportLastReadTime = useMemo(() => {
-        return ReportConnection.getReport(report.reportID)?.lastReadTime ?? report.lastReadTime ?? '';
-    }, [report.reportID, report.lastReadTime]);
+    const reportLastReadTime = report.lastReadTime ?? '';
```

Comment:
Can you please add tests specific to this change? to make sure we cover the case explicitly (the one for which this was added)